cmd/manager/aggregator.go

```diff
@@ -547,6 +547,12 @@ func createResults(crClient aggregatorCrClient, scan *compv1alpha1.ComplianceSca
 			cmdLog.Info("nil result or result.check, this shouldn't happen")
 			continue
 		}
+		_, hasHiddenLabel := pr.Annotations[compv1alpha1.RuleHideTagLabel]
```

Comment:
Here you are reading the `compv1alpha1.RuleHideTagLabel` value from `Annotations`...

Reply:
that's a good catch, let me fix it